src/SFA.DAS.LoginService.Web/Controllers/ChangeEmail/ConfirmChangeEmailController.cs

```diff
@@ -44,6 +49,13 @@ public async Task<IActionResult> ConfirmChangeEmail(Guid clientId, [FromForm] Co
             {
                 SetModelState(response, model);
                 return View(model);
+}
+
+            var authenticated = await HttpContext.AuthenticateAsync();
+
+            if (authenticated.Succeeded)
+            {
+                await HttpContext.SignInAsync(authenticated.Principal, authenticated.Properties);

```

Comment:
Probably can't happen, but if authenticated.Suceeded return false. We still go to ChnageEmailSuccessful page

Reply:
This addition isn't actually needed, I've removed it now.